libipl/p10/ipl0.C

```diff
@@ -550,6 +549,11 @@ static int ipl_updatehwmodel(void)
 	  	std::ofstream file(GENESIS_BOOT_FILE);
 	}
 
+	if(!apply_fco_override()) {
```

Comment:
Why you are calling aalway's this function? please refer my previous message related to function invoke.
"Add another commit to call this functiin update hw model
before process_guard_records();
if(!boot_file_absent ) {"

Reply:
Added as suggested
if(!boot_file_absent)
        apply_fco_override();